routers/router.go

```diff
@@ -0,0 +1,17 @@
+package routers
+
+import (
+	"challenge/api/controllers"
+
+	"github.com/gorilla/mux"
+)
+
+func Routes(router *mux.Router) {
+	controllers.InitNext()
+	router.HandleFunc("/", controllers.HomePage).Methods("GET")
```

Comment:
By convention, if you are using a hardcoded value more than 2 times, use a constant.
In this case you could use the `"net/http"` package, to use the Get constant.
`http.MethodGet`